source/includes/_api-reference.md

```diff
@@ -50,18 +58,59 @@ request.AddHeader("Authorization", "Basic [insert credentials here]");
 var request = new RestRequest(Method.GET);
 
 request.AddHeader("Authorization", "ApiKey [insert key here]");
+
+
+// Secure Access Token Authentication
+var request = new RestRequest(Method.GET);
+ 
+request.AddHeader("Authorization", "Bearer [insert Access token here]");
 ```
 
-Intelex utilizes two methods for authentication to the API- Basic and ApiKey.
+Intelex utilizes three methods for authentication to the API- Basic, ApiKey and Secure.
 
 Basic authentication is performed via HTTP Basic Auth and your Intelex user credentials (Username:Password). The credentials are then verified by Intelex and access will be granted or denied accordingly.
 
 ApiKey authentication can be performed by generating API access key(s) from the Intelex User Profile menu (note that this page is only accessible to full-access and admins on the site). 
 The unique generated key can then be used to access the Intelex REST API and used to authenticate and prove that the user is who they claim to be.
 
+Secure token based authentication is supported in V6 API. Only dedicated users can access the API.
+
+During the authentication process, the client provides the ‘client id’ and ‘client secret’ to the authentication endpoint, along with the audience and grant type, as demonstrated above. In response, the API furnishes an access token that comes with a specified time limit. This authentication is exclusively applicable to dedicated users. After successful authentication, users gain access to V6 API endpoints till the token expiration.
+
 All API requests must be made over HTTPS and API requests without authentication will fail. Security on data is managed by the platform security configuration. 
 API requests will maintain the same security settings you have configured in the platform for each user.
 
+>Example Requests:
+
+```CSharp 
+Body for Authentication Endpoint
+{"client_id":"Client ID Data",
+"client_secret":"Client Secret Data",
+"audience":"https://*.intelex.com/v6api",
```

Comment:
In my opinion it makes sense to make the audience like this: https://api.intelex.com/v6/

Reply:
Updated audience as "https://api.intelex.com/v6/"